src/backend/access/heap/rewriteheap.c

```diff
@@ -922,7 +887,7 @@ logical_heap_rewrite_flush_mappings(RewriteState state)
 					 errmsg("could not write to file \"%s\", wrote %d of %d: %m", src->path,
 							written, len)));
 		src->off += len;
-		wallog_mapping_file(src->path, FileGetRawDesc(src->vfd));
+		wallog_file_descriptor(src->path, FileGetRawDesc(src->vfd), -1);
```

Comment:
```suggestion
		wallog_file_descriptor(src->path, FileGetRawDesc(src->vfd), PG_UINT64_MAX);
```
Looks fragile to rely on the coercion from signed to unsigned here